tests/ctst/steps/iam-policies/IAMUser.ts

```diff
@@ -0,0 +1,48 @@
+import { Given, setDefaultTimeout } from '@cucumber/cucumber';
+import { Constants, IAM, Utils } from 'cli-testing';
+
+setDefaultTimeout(Constants.DEFAULT_TIMEOUT);
+
+/**
+ * Policy are using ARN, not names. This helper will dynamically extract a policy
+ * ARN from a CLI result
+ * @param {object} results - results from the command line
+ * @return {string} - the policy arn, or null if an error occured when
+ * parsing results.
+ */
+function extractPolicyArnFromResults(results: any) {
```

Comment:
Note for reviewers: removed in https://github.com/scality/Zenko/pull/1807, in favor of a more modular function